sky/skylet/constants.py

```diff
@@ -10,3 +10,6 @@
     'command to initialize it locally: sky launch -c {cluster} \'\'')
 
 JOB_ID_ENV_VAR = 'SKYPILOT_JOB_ID'
+
+SKYLET_VERSION = '1'
+SKYLET_VERSION_FILE = '.sky/skylet_version'
```

Comment:
Should be under `~/.sky/skylet_version`?
```suggestion
SKYLET_VERSION_FILE = '~/.sky/skylet_version'
```

If `~` is added, we need to have `os.path.expanduser` in the place we are using this constant.